topics/pytorch/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Facebook's AI Research lab (FAIR)
+display_name: PyTorch
+github_url: https://github.com/topics/pytorch
```

Comment:
This should be the URL of the PyTorch [organization or repository](https://github.com/github/explore/blob/main/docs/API.md#github_url), not the topic itself:

> (if applicable) URL of the topic's official GitHub organization or repository. Must start with `https://github.com/`.

Reply:
Sorry I've forgotten. I will update it.